sportstracker/src/main/resources/i18n/SportsTracker.properties

```diff
@@ -519,7 +519,7 @@ pv.main.recording_mode.text=Recording Mode
 pv.main.heartrate.text=Heart Rate:
 pv.main.altitude.text=Altitude:
 pv.main.speed.text=Speed:
-pv.main.cadence.text=Cadence:
+pv.main.cadence.text=Cadence-Step-Stroke:
```

Comment:
Could you please keep the wording "Cadence"? I think this term is generic enough, it's not just for cycling only. When we're adding "-Step-Stoke", than it's limited to this special types, maybe there are even more for other sport types - that's too much. Some for the labels below.

Reply:
Fine with me, I have changed that in both the English and German version